src/streams/openssl.h

```diff
@@ -13,6 +13,8 @@ extern int git_openssl_stream_global_init(void);
 
 extern int git_openssl_stream_new(git_stream **out, const char *host, const char *port);
 
+extern int git_openssl_set_cert_file(const char *file, const char *path);
```

Comment:
Some documentation would be nice here, even if it's private API. I certainly would be confused what "file" and "path" actually mean in this context